src/components/Escrow.js

```diff
@@ -1,41 +1,100 @@
+                let balance = await Escrow.getBalance();
+                setScanner('https://polygonscan.com');
+                setCount(ethers.utils.formatUnits(escrowCount, 0));
+                setContractData(['Polygon', lastEscrow, eip, ethers.utils.formatUnits(escrowCounters, 0), ethers.utils.formatUnits(balance, 18)]);
+                EscrowFactory.on('Launched', async (eip, escrow) => {
+                    console.log(eip, escrow);
+                    notify()
+                    escrowCount = await EscrowFactory.counter();
+                    escrowCounters = await EscrowFactory.escrowCounters(escrow);
+                    Escrow = new ethers.Contract(escrow, EscrowABI, provider)
+                    balance = await Escrow.getBalance();
+                    setScanner('https://polygonscan.com');
+                    setCount(ethers.utils.formatUnits(escrowCount, 0));
+                    setContractData(['Polygon', escrow, eip, ethers.utils.formatUnits(escrowCounters, 0), ethers.utils.formatUnits(balance, 18)]);
+                });
             } catch(err) {
-                console.error(err);
+                console.log("error_", err);
+                alert("Invalid escrow factory");
+            }
+
+            try {
+                const provider = getWeb3(networkMap['rinkeby'].rpcUrl);
+                const EscrowFactory = new ethers.Contract(networkMap['rinkeby'].defaultFactoryAddr, EscrowFactoryABI, provider)
+                let escrowCount = await EscrowFactory.counter();
+                let lastEscrow = await EscrowFactory.lastEscrow();
+                let eip = await EscrowFactory.eip20();
+                let escrowCounters = await EscrowFactory.escrowCounters(lastEscrow);
+                let Escrow = new ethers.Contract(lastEscrow, EscrowABI, provider)
+                let balance = await Escrow.getBalance();
+                setScanner('https://rinkeby.etherscan.io');
+                setCount(ethers.utils.formatUnits(escrowCount, 0));
+                setContractData(['Rinkeby', lastEscrow, eip, ethers.utils.formatUnits(escrowCounters, 0), ethers.utils.formatUnits(balance, 18)]);
 
+                EscrowFactory.on('Launched', async (eip, escrow) => {
+                    console.log(eip, escrow);
+                    notify()
+                    escrowCount = await EscrowFactory.counter();
+                    escrowCounters = await EscrowFactory.escrowCounters(escrow);
+                    Escrow = new ethers.Contract(escrow, EscrowABI, provider)
+                    balance = await Escrow.getBalance();
+                    setScanner('https://rinkeby.etherscan.io');
+                    setCount(ethers.utils.formatUnits(escrowCount, 0));
+                    setContractData(['Rinkeby', escrow, eip, ethers.utils.formatUnits(escrowCounters, 0), ethers.utils.formatUnits(balance, 18)]);
+                });
+            } catch(err) {
+                console.log("error_", err);
```

Comment:
DRY: you have 2 identical pieces of code(for different networks)
What we'd like to have - is to easy to manipulate configuration, where everyone can add a specific configuration(including addresses, scan url, rpc api url, chain id and etc.) and see changes in the UI